siege_utilities/identifiers/tests/test_uuid_generation.py

```
@@ -0,0 +1,215 @@
+        uuid5_from_seed(THING_NS, "   ")
+    with pytest.raises(ValueError, match="seed"):
+        uuid5_from_seed(THING_NS, "\t\n")
+
+
+def test_attestation_uuid_rejects_whitespace_only_inputs():
+    for bad_artifact, bad_parser, bad_values in [
+        ("   ", "v1", "h"),
+        ("a", "   ", "h"),
+        ("a", "v1", "   "),
+    ]:
+        with pytest.raises(ValueError):
```

Ruff (0.15.12)

[warning] `pytest.raises(ValueError)` is too broad, set the `match` parameter or use a more specific exception

(PT011)